torch/_torch_docs.py

```diff
@@ -6715,15 +6717,23 @@ def merge_dicts(*dicts):
         from the QR decomposition.
 
 Args:
-    input (Tensor): tensor of shape `(*, m, k)` where `*` is zero or more batch dimensions.
-    tau (Tensor): tensor of shape `(*, min(m, k))` where `*` is zero or more batch dimensions.
+    input (Tensor): tensor of shape `(*, mn, k)` where `*` is zero or more batch dimensions.
```

Comment:
What is `mn`? It has not been defined.
It might be interesting to write a full definition of what's going on at the beginning, as in
https://pytorch.org/docs/master/generated/torch.linalg.householder_product.html

It would even be reasonable to put this function in the `seealso` of `torch.linalg.householder_product`, as this function computes the action of the `householder_product` on a matrix (similar to `inv` vs `solve`) and it should often be preferred over `householder_product` + `matmul`.

Reply:
> What is mn? It has not been defined.

I'll change to
```
    input (Tensor): tensor of shape `(*, mn, k)` where `*` is zero or more batch dimensions
                    and `mn` is `m` or `n` depending on the :attr:`left`.
```